grpc-api/src/main/java/io/deephaven/grpc_api/arrow/FlightServiceGrpcImpl.java

```diff
@@ -238,11 +238,11 @@ public void doPutCustom(final InputStream request, final StreamObserver<Flight.P
             }
             final BarragePutMetadata app_metadata = mi.app_metadata;
             if (app_metadata == null) {
-                throw GrpcUtil.statusRuntimeException(Code.INVALID_ARGUMENT, "no app_metadata provided");
+                throw GrpcUtil.statusRuntimeException(Code.INVALID_ARGUMENT, "No app_metadata provided");
             }
             final ByteBuffer ticketBuf = app_metadata.rpcTicketAsByteBuffer();
             if (ticketBuf == null) {
-                throw GrpcUtil.statusRuntimeException(Code.INVALID_ARGUMENT, "no rpc ticket provided");
+                throw GrpcUtil.statusRuntimeException(Code.INVALID_ARGUMENT, "No rpc ticket provided");
```

Comment:
what's the goal in this? it isn't possible for rpcTicketAsByteBuffer() to return null